editors/emacs/files/site-start.el

```diff
@@ -10,3 +10,9 @@
 
 ;; Info-directory-list contains ${prefix}/share/info. See #32148.
 (setq Info-default-directory-list (cons "__PREFIX__/share/info" Info-default-directory-list))
+
+;; Use the OS X Emoji font for Emoticons
+(when (fboundp 'set-fontset-font)
+  (set-fontset-font "fontset-default"
+                    '(#x1F600 . #x1F64F)
```

Comment:
Doing this without choosing the `multicolor_font` variant may cause Emacs to crash.

`'symbol` may be better here, some emojis were not correctly displayed for me when using `'(#x1F600 . #x1F64F)`.